packages/kbn-pm/src/cli.ts

```diff
@@ -37,6 +37,8 @@ function help() {
         --skip-kibana-plugins   Filter all plugins in ./plugins and ../kibana-extra when running command.
         --no-cache              Disable the kbn packages bootstrap cache
         --no-validate           Disable the bootstrap yarn.lock validation
+        --force-yarn-install    Forces yarn install to run on bootstrap
+        --offline               Run in offline mode
```

Comment:
```suggestion
        --no-validate           Disable the bootstrap yarn.lock validation
        --force-install         Forces yarn install to run on bootstrap
        --offline               Run in offline mode
```

Reply:
👍 changed and committed the new kbn pm dist file